AWS Publish Cloud Account Id Images.yml

```diff
@@ -0,0 +1,105 @@
+trigger: none
+
+variables:
+  - group: Template Parameters
+  - group: Default Branch
+  - group: S3 uploads
+  - group: VPC ap-southeast-2
+  - group: VPC us-east-1
+  - group: Git Repos
+  - name: CookbooksSource1
+    value: c:\lansa
+  - name: imageReleaseState
+    value: Production
+  - name: jpnLanguage
+    value: JPN
+  - name: stackname
+    value: BakingDP-ImageRelease
+  - name: TemplateS3Namespace
+    value: /image-cd-pipeline
+
+resources:
+  pipelines:
+    - pipeline: _Build Cloud Account Id Artefacts
+      source: 'Build Cloud Account Id Artefacts  3.0'
+    - pipeline: _Build Image Release Artefacts
+      source: 'Build Image Release Artefacts  3.0'
+
+  repositories:
+    - repository: _LansaAWSTemplates
+      name: priyadishah/aws-templates-md
+      type: github
+      ref: support/scalable
+      endpoint: priyadishah
+    - repository: _priyadishahCookbooks
+      name: priyadishah/cookbooks
+      type: github
+      ref: feature/AWS-Cloud-Account-Id-Image-Release
+      endpoint: priyadishah
+
+pool:
+  vmImage: windows-2022
+
+stages:
+- template: templates/releaseTemplate.yml
```

Comment:
You are using the wrong directory and wrong template name. I've provided the correct one. If you can't find them talk to Nikhil or Tarun for details

Reply:
@robe070  give me a day to make above changes, I was helping others for release pipelines so couldn't make changes.

Reply:
For close to 2 weeks?

Reply:
i have fixed this and have pushed the code on my feature branch. 
![image](https://github.com/user-attachments/assets/74cacd93-725d-4009-934d-e89657582e1e)